src/pool/srv_target.c

```diff
@@ -533,6 +533,7 @@ pool_fetch_hdls_ult(void *data)
 			DP_UUID(pool->sp_uuid));
 		D_GOTO(out, rc);
 	}
+	memset(buf, 0, STACK_HDL_BUF_SIZE);
```

Comment:
This feels like an issue in the IV fetch implementation. buf looks like an OUT parameter, which the IV fetch implementation shouldn't read, no?

Reply:
yes, you are right. But I want to use pool_iv_ent_copy here, instead of adding another parameter to indicate it copyin/copy out.  And also since it is internal limited use api, so I just choose to initialize 0 here.

Reply:
OK.